### PR TITLE
Update developer setup instructions

### PR DESCRIPTION
**Pull request by sriramkrish85**

### CODE REVIEW

File: DEVELOPING.md [hunk 1]
```diff
@@ -22,8 +22,12 @@ We discuss implementation issues on [amphtml-discuss@googlegroups.com](https://g
 
 ### Installation
 
-1. `npm i`
-2. `edit /etc/hosts` and map `ads.localhost` and `iframe.localhost` to `127.0.0.1`:
+1. Install NodeJS from https://nodejs.org/
```
Comment (erwinmombay) [inline DEVELOPING.md]: we can also recommend `brew` (osx) or `apt`/`yum` on linux

Comment (jmadler) [inline DEVELOPING.md]: There are other installation mechanisms (e.g. brew) - let's not prefer one.  Just say "Install NodeJS" (you can link NodeJS to nodejs.org).

File: DEVELOPING.md [hunk 2]
```diff
@@ -22,8 +22,12 @@ We discuss implementation issues on [amphtml-discuss@googlegroups.com](https://g
...
+2. cd to /path-to-ampfolder/amphtml/
+3. run `npm i` command to install the required npm packages.
```
Comment (jmadler) [inline DEVELOPING.md]: Merge step 2 and 3 - say something like "In the repo directory, run ..."

File: DEVELOPING.md [hunk 3]
```diff
@@ -22,8 +22,12 @@ We discuss implementation issues on [amphtml-discuss@googlegroups.com](https://g
...
+4. run `sudo npm i -g gulp` command to install gulp in your local bin folder '/usr/local/bin/' on Mac.
+5. run `sudo npm i -g http-server` command to install http server in your local bin folder '/usr/local/bin/' on Mac.
```
Comment (jmadler) [inline DEVELOPING.md]: For step 4 and 5, lets not provide OS specific instructions - can you remove the sudo/mac bits?  Just say something like "install http-server by running "npm -i -g http-server".

Comment (jmadler) [inline DEVELOPING.md]: Also, is it possible to merge 4 and 5 into one command?

**Comment by sriramkrish85** [comment]
I signed it!

**Comment by cramforce** [comment]
LGTM

**Comment by erwinmombay** [comment]
@cramforce didnt see any of @jmadler recommendations applied, will there be a follow up?

**Comment by cramforce** [comment]
@jmadler Oh, shoot, I didn't see those.

**Comment by cramforce** [comment]
They were actually applied.

**Comment by erwinmombay** [comment]
weird, i still see the `sudo` stuff in the files. might be the merging that went on

**Comment by erwinmombay** [comment]
haha nvm, you already explained it :smile:

File: DEVELOPING.md (hunk 3 above)
Comment (cramforce) [inline DEVELOPING.md]: I'd keep the sudo. At least add "(might require sudo)". It'll be required in almost all unix-y systems.

File: DEVELOPING.md [hunk 4]
```diff
@@ -22,8 +22,11 @@ We discuss implementation issues on [amphtml-discuss@googlegroups.com](https://g
 
 ### Installation
 
-1. `npm i`
-2. `edit /etc/hosts` and map `ads.localhost` and `iframe.localhost` to `127.0.0.1`:
+1. Install [NodeJS](https://nodejs.org)
+2. In the repo directory and run `npm i` command to install the required npm packages.
```
Comment (jmadler) [inline DEVELOPING.md]: Bit of a typo on line 2

**Pull request merged**